Gemfile

```diff
@@ -4,3 +4,6 @@ source "http://rubygems.org"
 
 gemspec
 # This is an optional dev-dependency, required whenever sprockets is required
+gem "rubocop"
```

Comment:
why not in the gemspec as a development dependency?

Reply:
Because CI doesn't install development dependencies from the gemspec.

Do you want me to create a `Gemfile.development_dependencies` file like react_on_rails has?